uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-manage-entitlements-form.scss

```diff
@@ -0,0 +1,69 @@
+@mixin wider-element() {
+  margin-left: -20px;
+  margin-right: -20px;
+}
+
+@mixin column-side-borders() {
+  border-left: 1px solid $table-border-color;
+  border-right: 1px solid $table-border-color;
+}
+
+&.manage-entitlements-form {
+  .wider-hr {
+    @include wider-element();
+    border-width: 2px;
+  }
+
+  .wider-row {
+    @include wider-element();
+  }
+
+  .manage-entitlements-form__filters {
+    padding-bottom: 1em;
+  }
+
+  .table {
+    background: #FCFDFD;
+    margin-bottom: 2em;
+    border-top: 1px solid $table-border-color;
+    border-bottom: 1px solid $table-border-color;
+
+    th {
+      background: #F4F7F8 !important;
```

Comment:
Please try to use already defined colors from shoreditch,  those are already imported in https://github.com/civicrm/civihr/blob/c6f327a5a3d041898aa25d59f9c46d343ea1956c/uk.co.compucorp.civicrm.hrleaveandabsences/scss/overrides/_variables.scss#L9-L8